cmd/rpcdaemon/commands/tracing.go

```diff
@@ -113,15 +113,7 @@ func (api *PrivateDebugAPIImpl) traceBlock(ctx context.Context, blockNrOrHash rp
 		config.BorTraceEnabled = newBoolPtr(false)
 	}
 
-	var excessDataGas *big.Int
-	parentBlock, err := api.blockWithSenders(ctx, tx, block.ParentHash(), block.NumberU64()-1)
-	if err != nil {
-		stream.WriteNil()
-		return err
-	}
-	if parentBlock != nil {
-		excessDataGas = parentBlock.ExcessDataGas()
-	}
+	chainConfig, _ = api.chainConfig(tx)
```

Comment:
We already inited `chainConfig` at https://github.com/testinprod-io/op-erigon/blob/d44e9825a8087e6057d37abcc383c6a2e3a9f859/cmd/rpcdaemon/commands/tracing.go#L76
```suggestion
```